backend/fms_core/template_importer/row_handlers/library_qc/library_qc.py

```diff
@@ -84,11 +85,14 @@ def process_row_inner(self, sample_container, measures, process, process_measure
         if measures['concentration_nm'] is not None and measures['concentration_uL'] is not None:
             self.errors['concentration'] = 'Concentration must be specified in either nM or ng/uL, not both.'
 
+        # TODO: if it is a pool, library size is not provided. Should we force concentration in uL?
```

Comment:
This part should be moved bellow the update_library library size for non-pools. That way we can use uniformly the stored library_size (don't forget to update the object from db) for concentration calculations.